pages/fpki_cas.md

```diff
@@ -19,26 +17,22 @@ The FCPCA serves as the _root_ and _trust anchor_ for the _intermediate_ and _is
 
 - Federal Government Executive Branch Agencies  
 
-<br><br><br><br>
-
 {% include alert-info.html content="<strong>Public trust for websites</strong><br>A new effort is in the planning stages to establish another Federal Government root and issuing CAs dedicated to Public Trust Transport Layer Security (TLS) device certificates. Follow or contribute to the development of the Federal Government's new certificate policy for this Public Trust effort at https://github.com/uspki/policies" %}

```

Comment:
remove this too for now?

Reply:
I think we added this to avoid the info-alert box and the image from overlapping.  Removing may require tweaking the image padding (or tinkering with the CSS).